app/models/notifications/outgoing_contact_failed_notification.rb

```diff
@@ -5,15 +5,16 @@ def auth_mode
     end
 
     def absolute_user_ids
-      ids = if from_user.present?
-              [from_user.id]
-            else
-              []
-            end
-
-      ids.delete(to_user.id) if to_user.present?
-      ids = SecurityRole.admin_role.users.active.pluck(:id) if ids.blank?
-      ids.delete(to_user.id) if to_user.present?
+      ids = init_ids
+
+      if contact_log.sms?
+        ids.push(to_user.id) if to_user.present?
```

Comment:
basically, if it's an SMS that fails, just email the affected from/to user but not the admins